index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Static Web Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+
+<!-- CARD ONE -->
+<article>
+  <header>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|Share+Tech" rel="stylesheet">
+    <h1 class="productTitle">Q: Are We Not Men?</h1>
+  </header>
+
+
+<!-- IMG, DESCRIPTION, AVAILABILITY -->
+  
+  <section>
+    <img class="cover" src="devo.jpg"  height="300px" width="300px">
```

Comment:
Two things. First, the height and width HTML attributes should not have "px": http://www.w3schools.com/tags/att_img_height.asp

Second, don't do this. Use CSS to set height and width.